ProxyAgent/ProxyMiddleware.cs

```diff
@@ -31,6 +31,7 @@ public class ProxyMiddleware
             try
             {
                 await this.proxy.ProcessAsync(this.hostname, context.Request, context.Response);
+                this.log.Debug("--------------------------------------------------------------------------------", () => { });
```

Comment:
What are the dashes for?

Reply:
a debugging "helper" to distinguish one request from another, during development phases